tests/app/test_dependencies.py

```diff
@@ -38,6 +44,18 @@
     LiteratureSearchTool,
     MorphologyFeatureTool,
 )
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session
+
+
+def test_get_settings(monkeypatch):
```

Comment:
I think you forgot the `patch_required_env` here

Reply:
@BoBer78 
Added it.